CHANGELOG.md

```diff
@@ -76,6 +76,9 @@ Links "DE#nnn" prior to version 2.0 point to the Dash Enterprise closed-source D
   - allowed for functions, styleConditions and regular dictionaries to be passed to the `cellStyle` on all levels
   - added `rowId` to `cellClicked` data
 
+- [#111](https://github.com/plotly/dash-ag-grid/pull/111)
+  - fixing templates to only populate when `dangerously_allow_code=True`
```

Comment:
That note belongs under `Fixed` - but we should also note the addition of `filterModel` in the `Added` section

Reply:
adjusted via https://github.com/plotly/dash-ag-grid/pull/111/commits/679ed74b1af6dbeff7c142d63c62c7f845e893c7